### ISSUE TITLE
H2o xgboost R learning rate parameters bug

### ISSUE THREAD

**Issue by lescientifik** [h2o/h2o#1341]
Hi,

There seems to be an issue on R for xgboost implementation:
- Documentation for learn_rate and eta state that they are the same parameters, however, it is stated that they are given different default value (0 for eta and 0.1 for learn_rate
- When using one of this parameter with other values than 1, algorithm doesn't seem to do anything (loss is very high even for 0.9 as learn_rate, compared to 1).

Running on:

> R.version
               _                           
platform       x86_64-apple-darwin13.4.0   
arch           x86_64                      
os             darwin13.4.0                
system         x86_64, darwin13.4.0        
status                                     
major          3                           
minor          3.2                         
year           2016                        
month          10                          
day            31                          
svn rev        71607                       
language       R                           
version.string R version 3.3.2 (2016-10-31)
<img width="253" alt="learn_rate_0_9" src="https://user-images.githubusercontent.com/18367137/27554626-4d4418fe-5aaf-11e7-9a83-6c3bbd27ee18.png">
<img width="267" alt="learn_rate_1" src="https://user-images.githubusercontent.com/18367137/27554647-6323fdb0-5aaf-11e7-9860-cafe169270bd.png">

PS: running on latest H2O stable version (3.5.10.2)

**Comment by deweerdt** [comment]
Hello, @lescientifik this github repository is about an HTTP server. This might be the project you're looking for: https://github.com/h2oai/h2o-3

**Comment by lescientifik** [comment]
OOps, sorry for that, indeed that was what i was looking for... Thank you.
